doc/source/plugins.rst

```diff
@@ -0,0 +1,53 @@
+Plugins
+=======
+
+The Simphony library can extended through two `entry points`_ for
+contributing python modules that container engine and visualization components:
+
+- ``simphony.engine`` -- A python module that provides one or more
+  classes that implement the :class:`ABCEngine` interface.
+
+- ``simphony.visualization`` -- A python module that provides a simple
```

Comment:
I see that currently ``simphony.visualisation`` for the plugin but the current module name is ``visualization`` (``simphony/visualization/``).


I prefer **visualization** but can imagine that **visualisation** might be preferred